modules/ROOT/pages/osv2-guide.adoc

```diff
@@ -133,7 +133,8 @@ image::osv2-ui.png[]
 you can delete a partition or the store itself by hovering and clicking the trash
 can icon.
 
-*Note:* Object Store provides persistent storage of objects with a time to live (TTL) of 30 days.
+[NOTE]
+Object Store v2 provides persistent storage of objects with a time to live (TTL) of up to 30 days.
```

Comment:
Does "with a time to live..." modify storage or objects?

Reply:
Changed to: Objects are stored for up to 30 days, based on the TTL (time to live).